server/data/en/jf99.txt

```diff
@@ -169,23 +169,23 @@ Taking part in road traffic is potentially dangerous after donating blood.
 Emoticons are familiar to everyone nowadays.
 There is a social facility on the other side of the road.
 He lost a lot of money through gambling in the casino.
-A new movie is shown the first time in cinema.
+A new movie is shown for the first time in the cinema.
 This picture is pure art.
 A crowd of people stood around the fountain.
 When she entered the dance floor, everyone in the nightclub was looking at her.
 Wikipedia is a non-profit organization.
-The clock of the town center took damage from a lightening strike.
+The clock in the town center was damaged by a lightning strike.
 Water boarding is a terrible form of torture.
 The lawyer successfully defended his client.
-When the judge spoke the death sentence, the defendant showed no emotion.
+When the judge pronouced the death sentence, the defendant showed no emotion.
```

Comment:
You probably mean:
```suggestion
When the judge pronounced the death sentence, the defendant showed no emotion.
```

Reply:
Yes, you're quite right. Thanks for spotting that David!